src/style.scss

```diff
@@ -16,11 +16,10 @@
 
 :host {
   auro-dropdown {
-    --trigger-height: 40px;
-
     &::part(trigger) {
-      min-height: var(--trigger-height);
-      max-height: var(--trigger-height);
+      /* 2.5rem if --auro-size-500 not defined */
```

Comment:
nit-pick: this is SCSS, the recommended syntax for comments is `//` not `/**/` There is no need to persist these comments in the output CSS. 

Comments with `/**/` are typically for things like stylelint as this is doing a listing post-SCSS build.